components/script/dom/element.rs

```diff
@@ -147,6 +147,15 @@ pub enum ElementCreator {
     ScriptCreated,
 }
 
+impl ElementCreator {
+    pub fn is_parser_created(&self) -> u64 {
```

Comment:
The name of this method does not match what this method is actually doing. We should be returning a boolean here; the comment I made about extracting the line number belongs in a separate method.